yugabyte/src/yugabyte/ysql/client.clj

```diff
@@ -242,10 +246,9 @@
 (defmacro with-txn
   "Wrap evaluation within an SQL transaction using the default isolation level."
   [c & body]
-  `(j/with-db-transaction [~c ~c {:isolation isolation-level}]
+  `(j/with-db-transaction [~c ~c {:isolation txn-isolation-level}]
```

Comment:
Do we need to set isolation level it explicitly although it is set on connection level? Can we omit it ?

Reply:
We can - what it really does is `Connection.setTransactionIsolation`, the same way we do it